integrations/extensions/starter-kits/zendesk-guide/README.md

```diff
@@ -0,0 +1,141 @@
+
+## Other Setup Info
+
+### Setup in a new Assistant
+
+If you want to make a _new_ Assistant using this starter kit, take the following steps:
+
+- Download the OpenAPI specification (`zendesk-article-search-openapi.json`) and Actions JSON file (`zendesk-article-search-actions.json`) from the `basic` folder in this starter kit. (This section assumes the basic version of the starter kit, which will provide a basic Zendesk article search capability. For Zendesk filtered search, use the advanced version of the starter kit by downloading the files from the `advanced` folder.)
+- Use the OpenAPI specification to [build a custom extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension). In the `Import OpenAPI` step, you will upload (by click or drag-and-drop) the `zendesk-article-search-openapi.json` file to specify the authentication and methods for your extension. You will be able to review the list of the servers and server variables found within the OpenAPI document.
+- [Add the extension to your assistant](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-add-custom-extension) using the credentials you obtained in the first step above.
+  - For Authentication, select the authentication type provided in your openapi specification from the drop-down menu. In this starter kit, it is `Basic auth`.
+  - Next, enter the Zendesk username and password (typically {your_email}/token and the API token) that you obtained in the `Pre-Requisite Steps` section above.
+  - Finally, enter the value for your subdomain to access zendesk.com. For example, if your url is https://my-test-domain.zendesk.com, you would enter `my-test-domain` in the Server variables section.
+  - Review your extension setup and close to move on to the next steps.
+- Now you are ready to add the actions from the starter kit. Select `Actions` in the upper left menu. Then select `Global settings` at the top right of the Actions window and select the `Upload/Download` tab. Now [upload `zendesk-article-search-actions.json`](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import) to define the actions in this starter kit. Return to `Actions` and you will see three actions: `Search`, `Show search results`, and `Show search result` from the starter kit in the `Created by you` section.
+- Use either method listed in [Configuring Your Actions Skill to use an Extension](https://github.com/watson-developer-cloud/assistant-toolkit/blob/master/integrations/extensions/README.md#configuring-your-actions-skill-to-use-an-extension) to configure the actions you uploaded to invoke the custom extension you built.
+  - For the "Search" action, in the "Parameters" section of the "Use an extension" section (in "And then") the following parameter values:
+    - query = query_text
+    - per_page = page_limit
```

Comment:
I would add a note after the page_limit bullet saying something like:

Setting `per_page` is very important for the reliability of the search because Watson Assistant has a limit of 100kb on how much information can be saved in the session state.  For more details see [this note about the limits on the sizes of search results](https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/watson-discovery#limit-on-size-of-search-results).